saleor/graphql/account/mutations.py

```diff
@@ -1,9 +1,28 @@
 import graphene
 from django.contrib.auth.tokens import default_token_generator
+from django.contrib.sites.models import Site
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+from graphql_jwt.decorators import permission_required
 
-from ...account import models
+from ...account import emails, models
 from ...core.permissions import MODELS_PERMISSIONS, get_permissions
-from ..core.mutations import ModelMutation
+from ..core.mutations import BaseMutation, ModelMutation
+from ..core.types import Error
+
+
+def send_user_password_reset_email(user, use_https=False):
+    site = Site.objects.get_current()
+    context = {
+        'email': user.email,
+        'uid': urlsafe_base64_encode(force_bytes(user.pk)).decode(),
+        'token': default_token_generator.make_token(user),
+        'site_name': site.name,
+        'domain': site.domain,
+        'protocol': 'https' if use_https else 'http'
```

Comment:
In `settings.py` we have `ENABLE_SSL` setting that determines whether to use https, so the argument `use_https` could be dropped.